roles/edpm_nodes_validation/tasks/main.yml

```diff
@@ -63,11 +63,25 @@
     - edpm_nodes_validation_validate_controllers_icmp|bool
     - edpm_nodes_validation_ping_test_ips | length > 0
 
-- name: Verify the configured FQDN vs /etc/hosts
+- name: Verify a FQDN is set block
+  when:
+    - edpm_nodes_validation_check_for_fqdn|bool
+  block:
+  - name: Run hostname
+    command: hostname
+    register: hostname
+
+  - name: Verify a FQDN is set
+    ansible.builtin.assert:
+      that: hostname.stdout.find(".") != -1
+      fail_msg: "{{ hostname.stdout }} does not contain . and does not appear to be an FQDN."
+      success_msg: "{{ hostname.stdout }} contains . and appears to be an FQDN."
+
+- name: Verify the configured FQDN vs {{ edpm_nodes_validation_validate_fqdn_hosts_file }}
```

Comment:
Only thing that it might be worth checking is that /etc/hosts does not explicitly set the hostname or fqdn to the loopback address. Stackoverflow told everyone to do this a while back so might be done out of habit

Reply:
The task checking /etc/hosts already existed. It could be adoption relies on it, per 12f54de78cd98be81967beeaf4e98911af55f0cd @fao89 can you comment on that?

Reply:
actually, adotion@main doesn't need that, it was for adoption@dev-preview3
@jistr and @gibizer have more context on why we needed that for DP3